### PR TITLE
feat: prices GET endpoint

### PR DESCRIPTION
**Pull request by raphodn**
### What

- new `GET` endpoint `/prices`
- no authentication needed
- basic filters

### Todo (futur PRs)

- look at `fastapi-filter` for additional/complex filtering
- and `fastapi-pagination`

### CODE REVIEW

File: app/api.py [hunk 1]
```diff
@@ -124,6 +126,12 @@ async def authentication(form_data: Annotated[OAuth2PasswordRequestForm, Depends
     raise HTTPException(status_code=status.HTTP_500_INTERNAL_SERVER_ERROR, detail="Server error")
 
 
+@app.get("/prices", response_model=List[schemas.PriceBase])
```
Comment (raphodn) [inline app/api.py:129]: @raphael0202 this is a simple v1, but I feel we'll need to have improvements like 
- using [fastapi-filters](https://github.com/arthurio/fastapi-filter) to have more complex filtering
- pagination & limit parameters
- should the endpoint answer a list, or a dict with keys like "data" ?

Comment (raphael0202) [inline app/api.py:129]: we can use directly list[schemas.PriceBase] instead of List, as we're running on Python 3.10

Comment (raphael0202) [reply app/api.py:129]: pagination would be good to have indeed!
Better have a dict as a response, so we can add other fields if needed.
like `{"prices": [...]}`

Comment (raphodn) [reply app/api.py:129]: in Django Rest Framework the following format is returned
image

Comment (raphodn) [reply app/api.py:129]: I found [fastapi-pagination](https://github.com/uriyyo/fastapi-pagination), I'll add it in a follow-up PR

**Pull request merged**